game/glutil.cc

```diff
@@ -13,15 +13,6 @@ namespace glutil {
 		return result;
 	}
 		
-	VertexArray::VertexArray() {
-		glGenVertexArrays(1, &m_vao);
-		glGenBuffers(1, &m_vbo);
-	}
-
-	VertexArray::~VertexArray() {
```

Comment:
Don't delete this destructor, just the constructor.

Reply:
well it is empty now (just calling vector clear), thus useless IMHO

Reply:
It's clearning a vector of containing an array of objects consisting of several arrays.

Reply:
I don't see the point here: clear deletes all elements of vector, so does the destructor.

Reply:
*shrugs* I'm not gonna hold the PR back because of this but it seems odd to me you'd remove the `clear()` call but go to plenty of trouble to clean stuff up when the application exits.